app/models/manageiq/providers/kubevirt/infra_manager.rb

```diff
@@ -122,13 +123,20 @@ def self.verify_credentials(args)
   # @option opts [String] :token The Kubernetes authentication token.
   #
   def self.raw_connect(opts)
-    # Create the connection:
-    Connection.new(
-      :host      => opts[:server],
-      :port      => opts[:port],
-      :token     => ManageIQ::Password.try_decrypt(opts[:token]),
-      :namespace => "" # Collect resources across all namespaces
-    )
+    require "kubevirt"
+
+    api_key = ManageIQ::Password.try_decrypt(opts[:token])
+
+    kubevirt_api_client = Kubevirt::ApiClient.new
+    kubevirt_api_client.config.api_key    = api_key
+    kubevirt_api_client.config.scheme     = "https"
+    kubevirt_api_client.config.host       = "#{opts[:server]}:#{opts[:port] || 6443}"
+    kubevirt_api_client.config.verify_ssl = false # TODO check security_protocol
+    kubevirt_api_client.config.logger     = $log # TODO add a kubevirt logger
```

Comment:
I would expect the logger to come in from the opts otherwise we should probably default to a null logger (or just not set it)

Reply:
Yeah I just copied the use of `$log` from the existing Connection class, but per the note we should add a kubevirt logger

Reply:
https://github.com/ManageIQ/manageiq-providers-kubevirt/pull/279